lib/active_fedora/orders/association.rb

```diff
@@ -82,6 +82,12 @@ def delete_at(loc)
       target.delete_at(loc)
     end
 
+    # Delete all occurences of the specified target
+    # @param obj object to delete
+    def delete(obj)
```

Comment:
Why would deleting a proxy mess up unordered members? It's just a naming change - this association is in context of the proxies, not the proxyFor objects, and the methods should be named accordingly.